back-end/app.js

```diff
@@ -274,6 +274,7 @@ app.get('/shows-trakt/:id', (req, res, next) => {
         .then((responseA) => {
           //add response from Trakt API to final response object
           response_final = responseA.data
+          response_final['type'] = req.query.type
```

Comment:
Tests might need to be altered to reflect this change-- @Gkal2000 letting you know about this! Either you could add it into your PR and include this change in your tests, or if your tests get merged in before I finish this PR, I can add it into the tests.